src/Utility/Security.php

```diff
@@ -133,6 +133,18 @@ public static function randomBytes($length)
         return $bytes;
     }
 
+    /**
+     * Creates a secure random string.
+     *
+     * @param int $length String length
+     * @return string
+     * @since 3.6.0
+     */
+    public static function randomString($length)
+    {
+        return bin2hex(Security::randomBytes($length / 2));
```

Comment:
Doesnt the $length input here completely un-reflect the actual length returned?
If I say length 10, I would expect to get a string of length 6 (= 6 characters), so the length should be a substr() on the result of the bin2hex() before returning then IMO.

Reply:
Should you use `base64` encoding instead?